docs/access_rights/organization_account.md

```diff
@@ -8,53 +8,54 @@ paths:
-> If you are interested in joining the closed beta testing, contact us at [support@apify.com](mailto:support@apify.com?subject=Organization%20account%20beta%20testing).
+> The organization account is only available to accounts using our new billing system. If you see the migration message when you [log in](https://my.apify.com), you will need to migrate.
 
-Apify's organization account allows multiple team members to collaborate on projects. It enables you to manage your team members' [permissions]({{@link access_rights/list_of_permissions.md}}) and to centralize your billing.
+Organization accounts allow groups to collaborate on projects. It enables you to manage your team members' [permissions]({{@link access_rights/list_of_permissions.md}}) and to centralize your billing without having to share the credentials of a single personal account.
 
-At the same time, a separate organization account means teams no longer have to share the credentials of a single personal account. You can [switch]({{@link access_rights/organization_account/how_to_use.md#in-the-apify-app}}) between your personal and organization accounts seamlessly in just two clicks.
+You can [switch]({{@link access_rights/organization_account/how_to_use.md#in-the-apify-app}}) between your personal and organization accounts in just two clicks: in the [Apify app](https://my.apify.com), click the account button in the top-right corner, then select the organization.
 
 You can start an organization account in two ways.
 
-* [Convert an existing account](#convert-an-existing-account) into an organization. If your actors and integrations are set up in a personal account, it is probably best to convert that account into an organization. This will preserve all your integrations but means you will need a new personal account.
+* [Convert an existing account](#convert-an-existing-account) into an organization. If your actors and [integrations]({{@link tutorials/integrations.md}}) are set up in a personal account, it is probably best to convert that account into an organization. This will preserve all your integrations but means you will have a new personal account created for you.
 * [Create a new organization](#create-a-new-organization). If you don't have integrations set up yet, or if they are easy to change, you can create a new organization, preserving your personal account.
 
-## [](#create-a-new-organization) Create a new organization
+## [](#availability-and-pricing) Availability and pricing
+
+The organization account is available on our Team, Business, and Custom plans. [Visit our pricing page](https://apify.com/pricing) for more information.
 
-> **Warning**: the organization account is under closed beta testing. If you'd like to try it, contact us at [support@apify.com](mailto:support@apify.com?subject=Organization%20account%20beta%20testing).
+## [](#create-a-new-organization) Create a new organization
```

Comment:
I've noticed that the order of sections in switched, is that intentional? 
![image](https://user-images.githubusercontent.com/697301/105212234-7ea0aa00-5b4d-11eb-8e93-ae5acdfc2fb5.png)